src/lib/db/project-store.ts

```diff
@@ -112,6 +138,7 @@ class ProjectStore implements IProjectStore {
             id: row.id,
             description: row.description,
             health: row.health,
+            favorite: row.project !== null,
```

Comment:
this condition looks weird, why not row.favorite?

Reply:
Yes, lets do that!